src/components/GetStarted/Category/index.js

```diff
@@ -28,7 +28,7 @@ const Category = ({ classes, icon: Icon, title, description, tabs, rightAlign })
     >
       <Grid item xs={12} sm={6} md={6} lg={6} className={classes.CategoryContent}>
         <div className={classes.Title}>
-          {Icon & <Icon />}
+          <Icon />
```

Comment:
this will throw an error, when there is no Icon.. extract it out as gaurd clause